### PR TITLE
Modify query of containerized environments to get specific content

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to https://github.com/thoth-station/user-api/issues/1625

## This introduces a breaking change

- [x] No

## This should yield a new module release

- [x] Yes

## This Pull Request implements

Query containerized environments according to specific content (presence of python or RPM package, symbols...)

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm
thanks :+1:

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -2983,6 +2991,10 @@ def _construct_software_environments_query(
         python_version: Optional[str] = None,
         cuda_version: Optional[str] = None,
         image_name: Optional[str] = None,
+        library_name: Optional[str] = None,
+        symbol: Optional[str] = None,
+        python_package: Optional[list] = None,
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:2996]: can we use `package_name` to be consistent?

Comment (pacospace) [reply thoth/storages/graph/postgres.py:2996]: actually package names if this is a list

Comment (mayaCostantini) [reply thoth/storages/graph/postgres.py:2996]: Yes, and this is actually a `str`, thank you :+1:

File: thoth/storages/graph/postgres.py [hunk 2]
```diff
@@ -2889,6 +2889,10 @@ def get_software_environments_all(
         python_version: Optional[str] = None,
         cuda_version: Optional[str] = None,
         image_name: Optional[str] = None,
+        library_name: Optional[str] = None,
+        symbol: Optional[str] = None,
+        package_name: Optional[str] = None,
+        rpm_package: Optional[str] = None,
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:2895]: what about `rpm_package_name` to be more clear about the input?

Comment (mayaCostantini) [reply thoth/storages/graph/postgres.py]: Done (here and in https://github.com/thoth-station/user-api/pull/1628)

**Review by pacospace — APPROVED** [review]
lgtm! 💯 thanks!

**Pull request merged**